.expeditor/create_manifest.rb

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env ruby
+
+# By default, this script creates a manifest.json file that contains all the packages in the unstable channel.
+
+require 'date'
+require 'net/http'
+require 'json'
+require 'openssl'
+
+BLDR_API_HOST="willem.habitat.sh"
+BLDR_API_USER_AGENT="Chef Expeditor"
+
+# Packages that are present in
+# components/automate-deployment/pkg/assets/data/services.json but we wish to
+# exclude from the manifest (probably because they are not yet published to the
+# depot).
+#
+# We make this list explicit so that we can make manifest generation fail when
+# we fail to get expected package data from the hab depot.
+SKIP_PACKAGES = []
+
+def get_latest(channel, origin, name)
+  # TODO(ssd) 2018-07-23: Upgrading Habitat currently involves a
+  # restart of the entire process tree. Further, the Habitat project
+  # releases every 2 weeks, often giving us very little time to
+  # validate that the new release works as expected for us.
+  #
+  # IF YOU UPDATE THESE PINS YOU MUST ALSO UPDATE THE core/hab PIN IN components/automate-deployment/habitat/plan.sh
+  #
+  pinned_hab_components = {
+    "hab"          => { "origin" => "core", "name" => "hab",          "version" => "0.59.0", "release" => "20180712155441"},
+    "hab-sup"      => { "origin" => "core", "name" => "hab-sup",      "version" => "0.59.0", "release" => "20180712161546"},
+    "hab-launcher" => { "origin" => "core", "name" => "hab-launcher", "version" => "7797",   "release" => "20180625172404"}
+  }
```

Comment:
🤔 we're definitely going to want these updated to 0.64.1 for the Chef Server packages.
0.59.0 does not have the features we need.

Reply:
Those items were just cargo culted from the A2 version of this manifest. We don't really need them in this Chef Server iteration. In A2 the deployment service (which consumes this manifest) uses that data to know what version of the sup and launcher to install.

Reply:
In that case, could we remove these pins and just pick up the latest version of each component?